app/views/smart_proxies/welcome.html.erb

```diff
@@ -4,7 +4,7 @@
     <%= icon_text("sitemap", "", :kind => "fa") %>
   </div>
   <h1><%= _('Smart Proxies') %></h1>
-  <p><%= _("Smart proy provides restful API to subsystems such as DNS, DHCP, TFTP that Foreman talks to") %></p>
+  <p><%= _("Smart proxy provides restful API to subsystems such as DNS, DHCP, TFTP that Foreman talks to") %></p>
```

Comment:
"Smart proxies provide" sounds more natural, but I don't know why.

Reply:
It also ends in a preposition, which is a no no.